### ISSUE TITLE
REGRESSION: Can't open extension preferences page from Clippings Manager

### ISSUE THREAD

**Issue by aecreations** [aecreations/clippings-tb#131]
Clippings for Thunderbird 6.4a0+ on Thunderbird 102.0b5

An error occurs when attempting to open the extension preferences page from Clippings Manager (Tools -> Options). Error console output:

```
Uncaught (in promise) Error: An unexpected error occurred                                       undefined
    apply                  self-hosted:2669
    applySafeWithoutClone  resource://gre/modules/ExtensionCommon.jsm:704
    wrapPromise            resource://gre/modules/ExtensionCommon.jsm:952
```

Error does NOT occur when installed on Thunderbird 91.10.

**Comment by aecreations** [comment]
The line that opens the extension preferences page:
```
messenger.runtime.openOptionsPage();
```
... doesn't work on Thunderbird 102.0b5.

**Comment by aecreations** [comment]
Message posted on Thunderbird developers forum:
https://thunderbird.topicbox.com/groups/addons/T2aa8000cb1576f18/cant-open-extension-preferences-page-in-thunderbird-102-0b5

**Comment by aecreations** [comment]
Bugzilla bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1775223

**Comment by aecreations** [comment]
The above-referenced Bugzilla bug is fixed for Thunderbird 102.1.0. The workaround fix should only be used if the user is on Thunderbird version 102.0.x.

**Comment by aecreations** [comment]
Confirmed fixed on Thunderbird 103.0 beta 6.
Unable to confirm on Thunderbird 102.1.0 because it hasn't been released yet.

**Comment by aecreations** [comment]
Confirmed that this issue is no longer reproducible on Thunderbird 102.1.0.
